cnf-certification-test/operator/util/util.go

```diff
@@ -0,0 +1,10 @@
+package operator
+
+import (
+	"github.com/Masterminds/semver/v3"
+)
+
+func isValidSemanticVersion(version string) bool {
```

Comment:
There is already a function [IsValid()](https://pkg.go.dev/golang.org/x/mod/semver#IsValid) in the `semver` library to check if a version is valid. Also, I don't think we should do unit test of functions that belong to another library, so I suggest closing this PR and using this function wherever is needed.

Reply:
The function that you mentioned is part of golang.org/x/mod/semver package, not in the imported package I used. @sebrandon1 Do you think we shall use the package that Juan mentioned or use the already used one?

Reply:
@jmontesi I ran my unit test against your suggested package, but many test cases failed - this means that some valid semantic versions are not supported by the golang semver. So, in my opinion, using the package from Masterminds is better to use.